File: Kattis/CPP/IntroProblems/hissingmicrophone.cc
/*
kattis hissingmicrophone problem
Author: Agis Daniels
Solve given a line of input check if ss in in that string 
NOTE  
*/

#include <bits/stdc++.h>

using namespace std;

int main(){
    string s;
    cin>>s;
    if(s.find("ss")!=string::npos){
        cout<<"hiss"<<endl;
    }else{
        cout<<"no hiss"<<endl;
    }
    return 0;
}

File: Kattis/Python/IntroProblems/hissingmicrophone.py
""" 
kattis hissingmicrophone problem
Author: Agis Daniels
Solve given a line of input check if ss in in that string 
NOTE 
"""

from sys import stdin as rf
def main():
    ans=('ss' in rf.readline())
    if ans:
        print('hiss')
    else:
        print('no hiss')
if __name__=='__main__':
    main()
